packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -188,15 +188,7 @@ $btn-min-width: 26;
 
       ::slotted(#{$dds-prefix}-leadspace-heading),
       .#{$prefix}--leadspace__title {
-        @include content-width;
-
-        @include carbon--breakpoint(md) {
-          @include carbon--make-col(4, 6);
```

Comment:
I found these calls to `carbon--make-col` were ineffective on account of the parent (`.bx--leadspace__row`) being changed to a `flex-direction: column` on line 157. `carbon--mak-col` depends on `flex-basis` to set the width and since it gets changed to `flex-direction: column` it doesn't have the intended effect.

Reply:
@m4olivei Thanks for pointing this out! I wonder why it was adjusted to `column`. @ariellalgilmore Maybe you have some insight into this?

Ideally, we should stick to using the `carbon--mak-col` mixin which uses `flex-basis` to stay close to the `Carbon @2x grid` width as best as possible.

Reply:
The background behind the `flex-direction: column` change is here: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7004

Reply:
Thanks @ariellalgilmore . It feels like we could refactor to remove the `flex-direction: column` there and then get back to using `carbon--make-col`. The original change to make it `flex-direction: column` appears to have come in this PR: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/6154

I think if we want to do that, it would take some thought and more testing to do it without regressions. I'd suggest it's out of scope for this and could get handled in a follow up if there was interest.

Reply:
thanks! maybe we can add `TODO` statements on those lines, so we can come back to this.